app/services/analytics.rb

```diff
@@ -197,6 +197,7 @@ def browser_attributes
   IDV_GPO_VERIFICATION_SUBMITTED = 'IdV: GPO verification submitted' # Previously: "Account verification submitted"
   IDV_GPO_VERIFICATION_VISITED = 'IdV: GPO verification visited' # Previously: "Account verification visited"
   INVALID_AUTHENTICITY_TOKEN = 'Invalid Authenticity Token'
+  IN_PERSON_PROOFING = 'In Person Proofing'
```

Comment:
Hmm.  It's actually never called directly it's just part of the FSM and a base prefix.  I guess I can document it as such?

Reply:
Ideally we'd move away from that so we can document all the events 🤷

Reply:
How about I move it out of analytics.rb since it's technically not a singular event.  It is a prefix with a potentially infinite set of steps/events (submit, view) and the manual analytics logging is unnecessary.

Reply:
Sure, that works for now